randovania/gui/data_editor.py

```diff
@@ -145,6 +147,20 @@ def __init__(self, data: dict, data_path: Path | None, is_internal: bool, edit_m
                                           RandovaniaGame.METROID_PRIME_CORRUPTION}:
             self.area_view_dock.hide()
 
+        # values for dread and super metroid
+        min_zoom = 7
```

Comment:
I'd prefer that these values were to come from the game itself, not don't random constants here.
Maybe they're actually per-world?

Reply:
Yeah, that's pretty dirty. The range for the scale variable really depends on the size / resolution of the images.
Where is a good place for these constants? The `extra` section in the JSONs? Like `Artaria.json` etc?
Than I could change it to be set by the `on_select_world` method.

Reply:
Yeah, the extra section of each world is perfect.

Reply:
There's already some logic to get the image per world or per area depending on some magical config. You could hook the same events to decide if these values would be per-world or per-area.

Reply:
I now simply add the zoom from the slider to the already existing scale value. Not the best solution but still a little bit more comfort.